### DISCUSSION TITLE
Text missing from gui menu

### DISCUSSION THREAD

**Discussion by ChrisS1985ET** [Micke-K/IntuneManagement#77] [Q&A]
Hello all,

currently experiencing an issue where the text is not showing on the gui menus..  any ideas how i can fix this? this is on a new device i copied the files over from another. Attached screenshot.

![image](https://user-images.githubusercontent.com/110808012/183354348-cd403e59-2136-44a1-a3d5-6c327ecfce6d.png)

**Comment by Micke-K** [comment]
Hello!

I'll have a look and see if I can find a cause for it. 

Cheers

**Reply by Micke-K** [reply]
Confirmed bug. Thank you for reporting!

Should be fixed in 3.7.1

Cheers!
